src/blockchain/milestones.js

```diff
@@ -72,17 +72,10 @@ const createPaymentConversationAndSendEmail = async ({ app, milestone, txHash })
       },
       { performedByAddress: donations[0].actionTakerAddress },
     );
-    if (recipient && recipient.email) {
```

Comment:
I check the recipient in `donationsCollected` function, so it's not necessary to check in here